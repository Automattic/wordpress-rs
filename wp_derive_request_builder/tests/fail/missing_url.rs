#[derive(wp_derive_request_builder::WpDerivedRequest)]
enum UsersRequest {
    #[contextual_get(params = &UserListParams, output = Vec<SparseUser>)]
    List,
}

fn main() {}
