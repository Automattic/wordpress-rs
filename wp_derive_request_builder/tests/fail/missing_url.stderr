error: Missing (url = "/foo")
 --> tests/fail/missing_url.rs:4:7
  |
4 |     #[contextual_get(params = &UserListParams, output = Vec<SparseUser>)]
  |       ^^^^^^^^^^^^^^
